lesson15/index.js

```diff
@@ -0,0 +1,113 @@
+const div = document.getElementById("js-contents");
+const ul = document.getElementById("js-lists");
+
+function addLoading() {
+    const img = document.createElement("img");
+    img.src = "loading-circle.gif";
+    img.id = "js-loading";
+    div.appendChild(img);
+}
+
+function removeLoading() {
+    const img = document.getElementById("js-loading");
+    div.removeChild(img);
+}
```

Comment:
IMO
You can also write something like this
```suggestion
function removeLoading() {
    document.getElementById("js-loading").remove();
}
```

Reply:
Thank you for your review!!
I realized that I had been writing in a very complicated way...
I fixed it!
b27cd93